src/EasyOTA.cpp

```diff
@@ -185,13 +185,11 @@ int EasyOTA::connectWifi(unsigned long startTime, const String& wifi_ssid, const
 int EasyOTA::setupWifi(unsigned long now) {
 	static unsigned long retry_ms = 0;
 
-	if (_retries_current == 0)
+	if (now - retry_ms > 1000){
```

Comment:
Any particular reason timeout is not used here?

Reply:
before, there was following:
```
	if (_retries_current == 0)
		retry_ms = now;
	else {
		if (now - retry_ms <= 10 * _timeout)
			return 1;
	}
```
which means the function will run first time but then it will wait (return 1) until 10 * _timeout (100 seconds!) so the connection took 100 seconds minimum and no more retry was attempted because `_maxRetries = 0`

my code resets retry_ms timer if it was reached and runs the function. The connection retry time is not determined by `_timeout` because this variable is used in here https://github.com/foxis/EasyOTA/blob/master/src/EasyOTA.cpp#L180
(and its value (10 sec) is sane for the purpuse) but by the _maxRetries.